Makefile

```diff
@@ -196,9 +196,12 @@ verify:
 	@echo "verify: Started"
 	@echo "verify: Completed"
 
-test:
-	@echo "test: Started"
-	@echo "test: Completed"
+SKIP_TESTS ?=
+test: export LD_LIBRARY_PATH=$(VDDK_LIBS)
```

Comment:
Do we really need to export LD_LIBRARY_PATH in makefile?

Reply:
Move export LD_LIBRARY_PATH to test.sh